modules/survey_accounts/js/columnFormatter.js

```diff
@@ -0,0 +1,20 @@
+function formatColumn(column, cell, rowData) {
+    if (column === 'URL') {
+        var url = loris.BaseURL + "/survey.php?key=" + rowData[4];
```

Comment:
Can you update this to use the rowHeaders instead of a hardcoded? @alisterdev will show/explain how to do it.